typescript-forms/javascript-forms-quiz-notes.md

```diff
@@ -0,0 +1,54 @@
+# javascript-forms-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What event is fired when a user places their cursor in a form control?
+  focus
+- What event is fired when a user's cursor leaves a form control?
+  blur
+- What event is fired as a user changes the value of a form control?
+  input
+- What event is fired when a user clicks the `"submit"` button within a `<form>`?
+  submit
+- What does the `event.preventDefault()` method do?
+  prevents the default action that belongs to the event from occurring.
+- What does submitting a form without `event.preventDefault()` do?
+  will trigger the form's default action, which typically sends the form data to the server specified in the action attribute.
```

Comment:
Right. And if there is no action attribute, the browser will submit the form to itself which will refresh the page and update the url with the form values.